mdaviz/mda_folder_table_view_pager.py

```diff
@@ -0,0 +1,102 @@
+from functools import partial
+
+from PyQt5 import QtCore, QtWidgets
+
+class _AlignCenterDelegate(QtWidgets.QStyledItemDelegate):
+    """https://stackoverflow.com/a/61722299"""
+
+    def initStyleOption(self, option, index):
+        super().initStyleOption(option, index)
+        option.displayAlignment = QtCore.Qt.AlignCenter
+        
+class mdaFolderTableView(QtWidgets.QWidget):
```

Comment:
`MDAFolderTableViewPager`?  (as file name implies)

Reply:
This is the original file when I initially tried to make the table view work with the pager (directly recycled from `gemviz`). However the pager was making debugging complicated, so I decided to give up on the pager for now and come back to it later (issue #20). 
I renamed the file `[...]_pager.py` because it contained most of the pager code (what I had tried so far). Since I will need a pager for both the folder table view AND the data table view, I will eventually make a pager class (to be used for both table views), and rename that file accordingly.